File: C/Selection_sort.c
#include <stdio.h>
#include <stdlib.h>
void selection_sort(int arr[],int n);

int main()
{
    int arr[]={3,2,1,6,4,5,9,8,7};
    int n=sizeof(arr)/sizeof(arr[0]);
    selection_sort(arr,n);
    
}
void selection_sort(int arr[],int n)
{
    int min,temp;
    for(int i=0; i<n; i++)
    {
        min=i;
        for(int j=i+1; j<n; j++)
        {
            if(arr[j]<arr[min])
            {
                min=j;
            }

        }
        if(min!=i)
        {
            temp=arr[i];
            arr[i]=arr[min];
            arr[min]=temp;
        }
    }
    for(int i=0; i<n; i++)
    {
        printf("%d\t",arr[i]);
    }
}

File: Python/Selection_Sort.py
def selection_sort(arr,n):
    for i in range(0,len(arr)):
        min_idx=i
        for j in range (i+1,len(arr)):
            if arr[j]<arr[min_idx]:
                min_idx=j
        if(min_idx!=i):
            temp=arr[i]
            arr[i]=arr[min_idx]
            arr[min_idx]=temp
    for i in range(0,len(arr)):
        print("%d" % arr[i],end=" ")

l=list(map(int,input().split()))
n=len(l)
selection_sort(l,n)